src/coreclr/vm/interpexec.cpp

```diff
@@ -21,6 +21,12 @@ InterpThreadContext* InterpGetThreadContext()
         // FIXME VirtualAlloc/mmap with INTERP_STACK_ALIGNMENT alignment
         threadContext->pStackStart = threadContext->pStackPointer = (int8_t*)malloc(INTERP_STACK_SIZE);
         threadContext->pStackEnd = threadContext->pStackStart + INTERP_STACK_SIZE;
+        threadContext->pFrameDataAllocator = new FrameDataAllocator(INTERP_STACK_FRAGMENT_SIZE);
```

Comment:
Operator `new` will throw regular C++ out of memory exception. It does not work inside the interpreter since the intepreter is going to in a special exception handling regime. This should use `nothrow` new and check for null. (The other `new` in this method should be on the same plan.)

Reply:
This is a good point. The code is updated now and all `new` operators should be handled properly.